jovian/tests/utils/test_clone.py

```diff
@@ -265,3 +283,27 @@ def test_pull_get_latest_notebooks(mock_clone):
             call('aakashns/jovian-tutorial', None, fresh=False),
         ]
         mock_clone.assert_has_calls(calls, any_order=True)
+
+@pytest.mark.parametrize(
+    "content, result",
+    [
+        ( 
+            b"notebook content",
+            b"notebook content"
+        ),
+        (   
+            bytes(json.dumps({'metadata':{'key':'data','kernelspec':'data'}}),'utf-8'),
+            bytes(json.dumps({'metadata':{'key':'data'}}), 'utf-8')
+        )
+    ]
+)
+def test_sanitize_notebook(content, result):
+    assert _sanitize_notebook(content) == result
+
+def mock_bytes(*args, **kwargs):
+    raise TypeError('fake type error')
+
+@mock.patch("__main__.bytes", side_effect=mock_bytes)
+def test_sanitize_notebook_raises_exception(mock_bytes):
+        with pytest.raises(TypeError):
+            _sanitize_notebook(str("abc").encode('utf-8'))
```

Comment:
If you pass a string to _sanitize_notebook, it will throw JsonDecodeError(ValueError) in `json.loads` call and return back the string.

Reply:
string is changes to  `bytes(json.dumps({'metadata':{'key':'data','kernelspec':'data'}}),'utf-8')`